hnn_core/network.py

```diff
@@ -90,6 +91,147 @@ def _create_cell_coords(n_pyr_x, n_pyr_y, zdiff=1307.4):
     return pos_dict
 
 
+# connections:
+# this NODE is aware of its cells as targets
+# for each syn, return list of source GIDs.
+# for each item in the list, do a:
+# nc = pc.gid_connect(source_gid, target_syn), weight,delay
+# Both for synapses AND for external inputs
```

Comment:
is this where the comment was or should be? Seems a bit out of place

Reply:
Good point, this got dragged along when I was moving everything from `network_builder`, but it definitely doesn't make sense here any more. I'll put it back in an appropriate spot.